From akpm@linux-foundation.org  Wed Nov 12 11:31:08 2008
From: Damir N Abdullin <damir@mimas.ru>
Date: Thu, 30 Oct 2008 13:52:38 -0700
Subject: USB: cp2101: add Enfora GSM2228
Subject: + usb-serial-cp2101-add-enfora-gsm2228.patch added to -mm tree
To: mm-commits@vger.kernel.org
Cc: damir@mimas.ru, greg@kroah.com
Message-ID: <200810302052.m9UKqcAb002790@imap1.linux-foundation.org>


From: Damir N Abdullin <damir@mimas.ru>

Enfora GSM2228 based on Cygnal Integrated Products chip uses the same
cp2101 driver.

Signed-off-by: Damir N Abdullin <damir@mimas.ru>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/cp2101.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/cp2101.c
+++ b/drivers/usb/serial/cp2101.c
@@ -67,6 +67,7 @@ static struct usb_device_id id_table [] 
 	{ USB_DEVICE(0x10C4, 0x800A) }, /* SPORTident BSM7-D-USB main station */
 	{ USB_DEVICE(0x10C4, 0x803B) }, /* Pololu USB-serial converter */
 	{ USB_DEVICE(0x10C4, 0x8053) }, /* Enfora EDG1228 */
+	{ USB_DEVICE(0x10C4, 0x8054) }, /* Enfora GSM2228 */
 	{ USB_DEVICE(0x10C4, 0x8066) }, /* Argussoft In-System Programmer */
 	{ USB_DEVICE(0x10C4, 0x807A) }, /* Crumb128 board */
 	{ USB_DEVICE(0x10C4, 0x80CA) }, /* Degree Controls Inc */
